### ISSUE TITLE
406 response when creating a build with a branch name containing a .

### ISSUE THREAD

**Issue by npirotte** [nemerosa/ontrack#624]
```
[ontrack][request] GET https://ontrack.XXX.com/structure/entity/branch/XXX/XXX.js HTTP/1.1

[ontrack][response] GET https://ontrack.XXX.com/structure/entity/branch/XXX/XXX.js HTTP/1.1 CODE 406 

Error while accessing GET https://ontrack.XXX.com/structure/entity/branch/XXX/XXX.js HTTP/1.1: 406 -
```

**Comment by dcoraboeuf** [comment]
@npirotte , I took the liberty to adapt the path, because this error is very specific to the service.

**Comment by dcoraboeuf** [comment]
Fix available in 3.36.2
